sample_usage/print_nodes.py

```diff
@@ -21,40 +31,62 @@ def print_node(filename, node_class):
 
     for node in module.nodes_of_class(node_class):
         if node.fromlineno == node.end_lineno:
-            line = source_lines[node.fromlineno - 1]
-            out = [Style.DIM +
-                   line[:node.col_offset] +
-                   Style.BRIGHT + Fore.LIGHTBLACK_EX + Back.BLACK +
-                   line[node.col_offset:node.end_col_offset] +
-                   Style.DIM +
-                   line[node.end_col_offset:] + Style.RESET_ALL]
+            line = source_lines[node.fromlineno - 1]  # string
+            out = [
+                Style.DIM +
+                line[:node.col_offset] +
+
+                # The key part:
+                MAIN_COL +
+                line[node.col_offset : node.end_col_offset] +
+
+                Style.DIM + Fore.RESET + Back.RESET +
+                line[node.end_col_offset:]
+            ]
         else:
-            first_line = source_lines[node.fromlineno - 1]
-            last_line = source_lines[node.end_lineno - 1]
-            middle_lines = source_lines[node.fromlineno: node.end_lineno - 1]
-
-            out = (
-                [Style.DIM +
-                 first_line[:node.col_offset] +
-                 Style.BRIGHT + Fore.LIGHTBLACK_EX + Back.BLACK +
-                 first_line[node.col_offset:]] +
+            first_line = source_lines[node.fromlineno - 1]  # string
+            middle_lines = source_lines[node.fromlineno: node.end_lineno - 1]  # list
+            last_line = source_lines[node.end_lineno - 1]  # string
+
+            if middle_lines:
+                # For each item in the list of lines of strings,
+                # add colorama style to middle like the first and last lines
+                middle_lines = '\n'.join(list(map(wrap_color, middle_lines))) + '\n'
+            else:
+                middle_lines = ''  # coerce list to string
+
+            if first_line:  # Add a spacing after first_line
+                middle_lines = '\n' + middle_lines
+
+            out = [
+                Style.DIM + 
+                first_line[:node.col_offset] +
+                
+                # The key part:
+                MAIN_COL + 
```

Comment:
It is a good idea to have a helper function for the colouring like the one you created. I think you can use it here, and possibly other places too (with perhaps other modification).